Titanium.Web.Proxy/Network/CertificateManager.cs

```diff
@@ -219,6 +220,51 @@ public void TrustRootCertificate()
         }
 
         /// <summary>
+        /// Puts the certificate to the local machine's certificate store. 
+        /// Needs elevated permission. Works only on Windows.
+        /// </summary>
+        /// <returns></returns>
+        public bool TrustRootCertificateAsAdministrator()
+        {
+            if (RunTime.IsRunningOnMono())
+            {
+                return false;
+            }
+
+            var fileName = Path.GetTempFileName();
+            File.WriteAllBytes(fileName, RootCertificate.Export(X509ContentType.Pkcs12));
+
+            var info = new ProcessStartInfo
+            {
+                FileName = "certutil.exe",
+                Arguments = "-importPFX -Enterprise -p \"\" -f \"" + fileName + "\"",
```

Comment:
Telling the truth I don't know what is the difference. I tried it with -Enterprise argument and it did not add to the "Enterprise Trust". It added to the "Trusted Root CA".
Without the argument the result is the same. So you are right, it is better to remove that parameter.
I have a domain joined Windows 10 machine.

This solution works only on Windows, but if we make a small executable file which just imports the certificate and embedding it to TWP as an embedded resource, it can also work on another OS. What do you think about it? Would you mind if I create a small project and add the complied assembly to the TWP project as an embedded resource (of course the repository will contain only the source, so I won't upload harmful binary files:))?